products/sle/main.pm

```diff
@@ -973,15 +977,6 @@ elsif (get_var('HPC')) {
         elsif (check_var('HPC', 'hwloc') && sle_version_at_least('12-SP2')) {
```

Comment:
condition sle_version_at_least('12-SP2') can be removed, nobody runs hpc test on lower sle version